tests/run_packages_testing.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+http_proxy="${http_proxy:-}"
+https_proxy="${https_proxy:-}"
+DOCKERFILE_PATH="${SCRIPT_PATH}/Dockerfile"
+
+OS_DISTRIBUTION="fedora:31"
+
+install_packages() {
+	for i in "${OS_DISTRIBUTION[@]}"; do
+		echo "Test distribution packages for ${OS_DISTRIBUTION}"
+		run_test "${i}" "${DOCKERFILE_PATH}"
+		remove_image_and_dockerfile "${i}" "${DOCKERFILE_PATH}"
+	done
+}
+
+run_test() {
+	local OS_DISTRIBUTION=${1:-}
+	local DOCKERFILE_PATH=${2:-}
+	generate_dockerfile "${OS_DISTRIBUTION}" "${DOCKERFILE_PATH}"
+	build_dockerfile "${OS_DISTRIBUTION}" "${DOCKERFILE_PATH}"
```

Comment:
OOI, do we intend to run any actual tests after we've done the dockerfile build?